docs/platform/howto/byoc/delete-custom-cloud.rst

```diff
@@ -0,0 +1,55 @@
+Delete your custom cloud
+========================
+
+.. topic:: Custom clouds
+
+    A :doc:`custom cloud </docs/platform/concepts/byoc>` is your own cloud infrastructure integrated with your Aiven organization. Using a custom cloud in Aiven may be the optimal solution if you have specific business needs or project requirements, such as a strict regulatory compliance. Creating and using custom clouds in your Aiven organization requires :doc:`enabling the bring your own cloud (BYOC) feature </docs/platform/howto/byoc/enable-byoc>`.
+
+This article details how to delete your custom cloud in `Aiven Console <https://console.aiven.io/>`_.
+
+About deleting custom clouds
+============================
+
+After deleting a custom cloud, the data co-hosted in this cloud will no longer be available from the Aiven platform. The deletion only impacts resources on the Aiven site (such as cloud configuration files and the bastion service) and no cloud resources on your AWS account are touched. Make sure there are no active services using your custom cloud before deleting this cloud.
```

Comment:
This is not quite correct, because the bastion service does run in the customer's cloud account, and we do delete it when deleting a custom cloud, which means deleting the EC2 instance.

What does remain after a custom cloud deletion are the resources created when applying the Terraform template when the custom cloud was created. Removing them would involve running [terraform destroy](https://developer.hashicorp.com/terraform/cli/commands/destroy) with the same template and variables file that were used for creating them.